packages/@monocdk-experiment/assert/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@monocdk-experiment/assert",
+  "version": "1.25.0",
+  "description": "An assertion library for use with CDK Apps",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "build": "./clone.sh && cdk-build",
+    "watch": "cdk-watch",
+    "lint": "cdk-lint",
+    "test": "cdk-test",
+    "pkglint": "pkglint -f",
+    "package": "cdk-package",
+    "build+test+package": "npm run build+test && npm run package",
+    "build+test": "npm run build && npm test"
+  },
+  "jest": {
+    "coverageThreshold": {
+      "global": {
+        "statements": 75,
+        "branches": 65
+      }
+    }
+  },
+  "author": {
+    "name": "Amazon Web Services",
+    "url": "https://aws.amazon.com",
+    "organization": true
+  },
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@types/jest": "^25.1.2",
+    "cdk-build-tools": "1.25.0",
+    "jest": "^24.9.0",
+    "pkglint": "1.25.0",
+    "ts-jest": "^25.2.0"
+  },
+  "dependencies": {
+    "@aws-cdk/cloudformation-diff": "1.25.0",
+    "monocdk-experiment": "1.25.0"
```

Comment:
This needs to be a peerDependency on `^1.25.0`.

Reply:
Fixed